wsagent/che-core-api-project/src/test/java/org/eclipse/che/api/project/server/ProjectServiceTest.java

```diff
@@ -288,6 +278,27 @@ public void setUp() throws Exception {
         env = org.eclipse.che.commons.env.EnvironmentContext.getCurrent();
     }
 
+    private void addMockedProjectConfigDto(org.eclipse.che.api.project.server.type.ProjectTypeDef myProjectType, String projectName)
+            throws ForbiddenException, ServerException, NotFoundException, ConflictException {
+        final ProjectConfigDto testProjectConfigMock = mock(ProjectConfigDto.class);
+        when(testProjectConfigMock.getPath()).thenReturn("/" + projectName);
+        when(testProjectConfigMock.getName()).thenReturn(projectName);
+        when(testProjectConfigMock.getDescription()).thenReturn("my test project");
+        when(testProjectConfigMock.getType()).thenReturn("my_project_type");
+        when(testProjectConfigMock.getSource()).thenReturn(DtoFactory.getInstance().createDto(SourceStorageDto.class));
+        //        when(testProjectConfigMock.getModules()).thenReturn(modules);
```

Comment:
I just extracted the sources into a separate method and those comments already were there. Unfortunately there is no information about the author of that ugly code so I'm still not sure if that was left for some reason or by a mistake.

As I'm quite a new person to this very project I feel some responsibility for the legacy sources, so I think that it is yet not of my competence to remove someones comments.

Anyway if you feel confident and recommend to remove it - that should not be a problem at all ;-)